diff --git a/files/files.go b/files/files.go
index 7b220fe..f0c9c6b 100644
--- a/files/files.go
+++ b/files/files.go
@@ -193,10 +193,6 @@ func appendGlobbedFiles(all Contents, globbed map[string]string, origFile *Conte
 			FileInfo:    newFileInfo,
 			Packager:    origFile.Packager,
 		}).WithFileInfoDefaults()
-		if dst, err := os.Readlink(src); err == nil {
-			newFile.Source = dst
-			newFile.Type = "symlink"
-		}
 
 		all = append(all, newFile)
 	}
diff --git a/rpm/rpm.go b/rpm/rpm.go
index d9559ba..a2bbd42 100644
--- a/rpm/rpm.go
+++ b/rpm/rpm.go
@@ -253,7 +253,7 @@ func formatVersion(info *nfpm.Info) string {
 	version := info.Version
 
 	if info.Prerelease != "" {
-		version += "~" + info.Prerelease
+		version += "-" + info.Prerelease
 	}
 
 	if info.VersionMetadata != "" {
